### ISSUE TITLE
Add more examples to Readme

### ISSUE THREAD

**Issue by BitcoinZavior** [LtbLightning/bdk-rn#11]
The readme can do with a few variations of the `createDescriptor`, `createWallet`, `generateMnemonic` and transaction related methods.

**Comment by ibaddesmukh** [comment]
Can I help out with this?

**Comment by BitcoinZavior** [comment]
@ibaddesmukh yes you can. Let me know if you need any info or help to get you started.
